helm_drydock/drivers/node/maasdriver/api_client.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2017 AT&T Intellectual Property.  All other rights reserved.
```

Comment:
Required no, but their client is focused on Async I/O while drydock is using a multithreading pattern. And since they are missing quite a bit of the API functionality that Drydock will utilize, I thought it would be easier to just utilize the API directly. The client can be used synchronously but its interface isn't well documented and isn't natural to me.